code/style.css

```diff
@@ -1,4 +1,273 @@
-html {
-  background: #0025ff;
-  color: #fff;
+body {
+  background-color: #ffffff;
+  color: #202121;
+  font-family: "Montserrat", sans-serif;
+  margin: 0;
+}
+
+.container {
+  margin: 5%;
+}
+
+/* Header */
+
+header {
+  padding: 20px 5%;
+  box-shadow: 0 -50px 25px 50px #c9c9c9;
+}
+
+.flex-header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.flex-navbar {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-end;
+}
+
+.flex-child {
+  margin-left: 18%;
+}
+
+a {
+  text-decoration: none;
+  color: #202121;
+  text-transform: uppercase;
+}
+
+.navbar {
+  font-size: 1rem;
+  font-weight: 300;
+  line-height: 1.75rem;
+}
+
+.navbar-brand {
+  font-size: 1.3rem;
+  font-weight: 800;
+}
+
+.navbar:hover {
+  color: #bbbbbb;
+}
+
+/* Paragraphs and Headlines */
+
+h1 {
+  font-size: 1.5rem;
+  margin: 0 0 10px 0;
+}
+
+h2 {
+  font-size: 1rem;
+  margin: 10px 0 10px 0;
+}
+
+p {
+  font-size: 0.875rem;
+  font-weight: 300;
+  line-height: 1.75rem;
+  margin: 10px 0 15px 0;
+}
+
+.hero-text p {
+  font-size: 1rem;
+  line-height: 2rem;
+  margin-bottom: 25px;
+}
+
+/* Footer & Footer-Button */
+
+footer {
+  margin: 0;
+}
+
+.footer-button {
+  padding: 0% 5% 5% 5%;
+  text-align: right;
+}
+
+.fa-arrow-alt-circle-up {
+  font-size: 2.5rem;
+  color: #d8d8d8;
+  justify-content: flex-end;
+}
+
+.fa-arrow-alt-circle-up:hover {
+  color: #202121;
+}
+
+/* Images and Grid */
+
+.hero-image {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.small-news-images {
+  object-fit: cover;
+  width: 100%;
+  height: 61vw;
+}
+
+.small-news-section,
+.hero-article {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: 40px;
+}
+
+.hero-section {
+  margin-bottom: 40px;
+}
+
+.hero-image-container,
+.hero-text,
+.small-articles {
+  grid-column: span 4;
+}
+
+img:hover,
+h1:hover,
+h2:hover,
+p:hover {
+  cursor: pointer;
+}
+
+/* Responsiveness and Breakpoints */
+
+/* Specification for mobile */
+@media (min-width: 0px) and (max-width: 767px) {
+  .hidden-paragraph,
+  .hidden-paragraph-2 {
+    display: none;
+  }
+  .hero-article {
+    grid-gap: 20px;
+  }
+  .container {
+    margin-top: 40px;
+  }
+}
```

Comment:
Great that you've built your page for mobile view, and then added code for tablet and desktops and not the other way around!